### ISSUE TITLE
Add F18A VDP firmware compatibility mode to Classic99!

### ISSUE THREAD

**Issue by TheRealHamtaro126** [tursilion/classic99#8]
If possible, Can you add an option to select the F18A, which is a video chip used in both "visually boosted"/enhanced TI99/4A's and the CollectorVision Phoenix console (which is actually a ColecoVision clone rather than a MSX or TI99/4A clone since they share the same video hardware with a different revision sometimes.) Please?

It adds several features including enhanced 256-color palette, better tiles, better detailed sprites, and even more. and on the plus side is it is backwards compatible with all classic TI video modes.

Here is the site for F18A development:
https://dnotq.io/f18a/f18a.html

**Comment by tursilion** [comment]
I helped Matt test the F18A, and I wrote the ColecoVision menu system 
for the Phoenix. I know what they are. ;)

Classic99 has supported the F18A's GPU for the last five years or so. 
For the last couple of years it has supported the palette configuration, 
enhanced sprites, 80 column mode and dual layers. It does not support 
the bitmap layer, 30 row mode, or scroll registers.

It's on by default :)



TheRealHamtaro126 wrote on 2021-11-21 6:58 PM:
